project-koku/koku
# [COST-3891] Worker for SWATCH Data Extraction

## Jira Ticket

[COST-3891](https://issues.redhat.com/browse/COST-3891)

## Description

This change will add the initial set up for SWATCH data extraction task worker.

## Testing

1. Checkout Branch
2. Restart Koku
3. Load test data
    ```
    make create-test-customer
    make load-test-customer-data
    ```
4. Hit the [celery_queue_lengths](http://127.0.0.1:5042/api/cost-management/v1/celery_queue_lengths/) endpoint.
    - You should see the subs extraction in the queues returned.
    ```
    {
        "download": 0,
        
        ...

        "subs_extraction": 0
    }


## Notes

...
